Registry/Registry.NMM

```diff
@@ -1213,6 +1215,23 @@ state integer   pgsxy      ij      -       1      -     rh
 
 state    real    gecros_state i{gecros}j  -       1      Z     rhd=(interp_mask_land_field:lu_index)u=(UpNear)     "gecros"      "gecros state array"         "-"
  
+#Noah-MP Irrigation
```

Comment:
Is it true that we need these items from the noahmp registry also in Registry.NMM. Doesn't it just include the NoahMP registry? We need to mimic how other NoahMP variables are handled if NMM supports NoahMP.

Reply:
answering my own comment, yes, the noahmp registry variables are duplicated in the NMM Registry. Seems like an include could replace a lot of duplication, but that is beyond this PR and to do later.

Reply:
The problem is that the interpolation/feedback keyword part is different for EM and NMM.